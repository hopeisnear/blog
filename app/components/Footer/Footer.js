import React from 'react';

import './Footer.scss';

export default function Footer() {
  return (
    <footer className="Footer">
      <section>
        <div className="footer__note">
          Made with
          <span role="img" aria-label="heart-emoji">❤️</span>
          by Rafal Szczepankiewicz
        </div>
      </section>
    </footer>
  );
}
