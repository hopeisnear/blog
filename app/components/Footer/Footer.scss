@import "../../styles/variables.scss";

.app-footer {
  border-top: 1px solid $main-border-color;
  .footer-note {
    padding-top: 20px;
    padding-bottom: 20px;
  }
}
