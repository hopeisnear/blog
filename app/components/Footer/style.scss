.app-footer {
  .footer-note {
    padding-top: 50px;
    padding-bottom: 50px;
    height: auto;
    max-width: 1280px;
    margin: 0 auto;
    padding-left: 25px;
    padding-right: 25px;
    background: #cccccc;
  }
}
