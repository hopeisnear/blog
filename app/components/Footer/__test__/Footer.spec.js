import React from 'react';
import { shallow } from 'enzyme';

import Footer from '../Footer';

describe('<Footer />', () => {
  test('should render <Footer />', () => {
    const footerComponent = shallow(<Footer />);

    expect(footerComponent).toMatchSnapshot();
  });
});
