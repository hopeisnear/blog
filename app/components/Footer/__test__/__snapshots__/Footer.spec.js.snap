// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render <Footer /> 1`] = `
<footer
  className="Footer"
>
  <section>
    <div
      className="footer__note"
    >
      Made with
      <span
        aria-label="heart-emoji"
        role="img"
      >
        ❤️
      </span>
      by Rafal Szczepankiewicz
    </div>
  </section>
</footer>
`;
